### PR TITLE
Work around geckojs build bustage from UniFFI; fixes JIRA:SDK-87

### PR DESCRIPTION
**Pull request by dmose**

### CODE REVIEW

**Review by eoger — APPROVED** [review]
It's hacky, but the issue shouldn't block us.

**Comment by dmose** [comment]
The original patch didn't build everywhere, because the changes reflected in the IDL weren't propagated through to the hand-coded stuff in the various bindings.  After chatting with @mhammond, it seemed to make more sense to simply make client_id not be optional, since that's presumably where it wants to end up anyway.  Sorry if this is the wrong way forward; wanted to unblock folks working on gecko...

**Comment by dmose** [comment]
Bah, not compiling on Circle.  I'll have to look more to see why it's building locally for me but not here..

File: nimbus/src/evaluator.rs [hunk 1]
```diff
@@ -71,7 +71,7 @@ pub fn evaluate_enrollment(
         match available_randomization_units
```
Comment (mhammond) [inline nimbus/src/evaluator.rs:71]: We no longer need this match - ie, this line can read `let id = available_randomization_units.get_value(...);`

**Review by mhammond — CHANGES_REQUESTED** [review]
LGTM, and I agree that longer term there's no need to support consumers who can't supply a `client_id` - but we should clean up that one block.

File: nimbus/src/evaluator.rs (hunk 1 above)
Comment (dmose) [reply nimbus/src/evaluator.rs]: Fixed.

**Review by mhammond — APPROVED** [review]

**Pull request merged**